hack/get-machine-run-tests.sh

```diff
@@ -96,22 +116,29 @@ if echo "${EXTRA_ANSIBLE_OPENSHIFT:-}" | grep -q -i "use_crio=true" ; then
 else
     USE_CRIO=${USE_CRIO:-false}
 fi
-export REMOTE_GOPATH=${REMOTE_GOPATH:-/data}
+GOPATH=${GOPATH:-$HOME/go}
```

Comment:
we use `go` instead of `golang` ?

Reply:
I thought the convention was to use `$HOME/go`?  https://golang.org/doc/code.html#GOPATH
" The GOPATH environment variable specifies the location of your workspace. It defaults to a directory named go inside your home directory, so $HOME/go on Unix, $home/go on Plan 9, and %USERPROFILE%\go (usually C:\Users\YourName\go) on Windows. "

Reply:
oh, I may just not be following the convention then.